README.md

```diff
@@ -78,19 +78,33 @@ yourlogger.addHandler(airbrake.AirbrakeHandler())
 _by default, the `AirbrakeHandler` only handles logs level ERROR (40) and above_
 
 #### Additional Options
-More options are available to configure this library. Similar to setup, you can do this with environment variables
+More options are available to configure this library. 
+
+For example; to add more context to your errors, you can send the hostname. One way is by setting the HOSTNAME env var.
```

Comment:
* Please wrap to 80 characters.
* Not sure about the semicolon, probably should be a comma?